app/editor/src/features/admin/event-of-the-day/hooks/useColumns.tsx

```diff
@@ -103,6 +103,28 @@ export const useColumns = (
     }
   };
 
+  const formatOptionLabel = (data: ITopicOptionItem) => (
+    <div
+      className={
+        (data.value === topicIdNotApplicable ? `type-not-applicable` : `type-${data.topicType}`) +
+        // This extra style exists only to flag disabled topics that are disabled.
+        // These could show up because of migration from TNO, or through changes to
+        // content and topics that are possible
+        (data.isDisabled ? ' type-disabled' : '')
+      }
+    >
+      <span
+        className={
+          `option-hint ` +
+          (data.value === topicIdNotApplicable ? `type-not-applicable` : `type-${data.topicType}`)
+        }
+      >
+        {data.topicType.charAt(0)}
+      </span>
+      {data.label}
+    </div>
+  );
+
```

Comment:
format topic option items in the list with an indicator to make it clear whether they are an Issue or Proactive topic
![image](https://github.com/bcgov/tno/assets/63015960/41fcf8f1-bd1a-4796-b79c-b3bc373b59af)